src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1116,8 +1117,54 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger? errorLogger, Cancella
             }
 
             compilation.GetDiagnostics(CompilationStage.Declare, includeEarlierStages: false, diagnostics, cancellationToken);
+
+            // If there are unsuppressable declaration errors, we want to exit early from this method.
+            // But before we do so, we need to run diagnostic suppressors (if any).
             if (HasUnsuppressableErrors(diagnostics))
             {
+                if (analyzerDriver == null || !analyzerDriver.HasDiagnosticSuppressors)
+                {
+                    return;
+                }
+
+                string outputFileName = GetOutputFileName(compilation, cancellationToken);
+                string pdbFilePath = Arguments.GetPdbFilePath(outputFileName);
+                var emitOptions = GetEmitOptions(outputFileName, pdbFilePath);
+
+                diagnostics.Clear();
+                var moduleBuilder = 
```

Comment:
Probably the extra space here is what causes CI fail. 

```suggestion
                var moduleBuilder =
```

Reply:
Thanks, fixed: https://github.com/dotnet/roslyn/pull/62641/commits/f10c564fbda6291cc605323d832f0a3269ac5762